codex-cli/src/approvals.ts

```diff
@@ -112,23 +117,23 @@ export function canAutoApprove(
     } catch (e) {
       // In practice, there seem to be syntactically valid shell commands that
       // shell-quote cannot parse, so we should not reject, but ask the user.
-      switch (policy) {
```

Comment:
This should have stayed a `switch` statement because we have ESLint rules set up to ensure `switch` statements are exhaustive so that if a new variant of the `ApprovalPolicy` enum is introduced we are forced to address the existing callsites.